src/main/java/frc/robot/Robot.java

```diff
@@ -62,40 +65,44 @@ public Robot() {
         autoChooser = AutoBuilder.buildAutoChooser();
     }
 
-    @Override
-    public void robotInit() {
-        super.robotInit();
-
-        // Record metadata
-        Logger.recordMetadata("ProjectName", BuildConstants.MAVEN_NAME);
-        Logger.recordMetadata("BuildDate", BuildConstants.BUILD_DATE);
-        Logger.recordMetadata("GitSHA", BuildConstants.GIT_SHA);
-        Logger.recordMetadata("GitDate", BuildConstants.GIT_DATE);
-        Logger.recordMetadata("GitBranch", BuildConstants.GIT_BRANCH);
-        switch (BuildConstants.DIRTY) {
-            case 0:
-                Logger.recordMetadata("GitDirty", "All changes committed");
-                break;
-            case 1:
-                Logger.recordMetadata("GitDirty", "Uncomitted changes");
-                break;
-            default:
-                Logger.recordMetadata("GitDirty", "Unknown");
-                break;
-        }
-
-        map.setupLogging();
-        if (!isReal()) {
-            setUseTiming(false); // Run as fast as possible
-        }
-        // Start logging! No more data receivers, replay sources, or metadata values may
-        // be added.
-        Logger.start();
-
-        led.colorAlliance().schedule();
-        DriverStation.silenceJoystickConnectionWarning(true);
-
-    }
+    // Logging stuff that we don't need yet. Do we need it at all for Alpha? - Tim
+    // 1/10 3:45
+
+    // @Override
+    // public void robotInit() {
+    // super.robotInit();
+
+    // // Record metadata
+    // Logger.recordMetadata("ProjectName", BuildConstants.MAVEN_NAME);
+    // Logger.recordMetadata("BuildDate", BuildConstants.BUILD_DATE);
+    // Logger.recordMetadata("GitSHA", BuildConstants.GIT_SHA);
+    // Logger.recordMetadata("GitDate", BuildConstants.GIT_DATE);
+    // Logger.recordMetadata("GitBranch", BuildConstants.GIT_BRANCH);
+    // switch (BuildConstants.DIRTY) {
+    // case 0:
+    // Logger.recordMetadata("GitDirty", "All changes committed");
+    // break;
+    // case 1:
+    // Logger.recordMetadata("GitDirty", "Uncomitted changes");
+    // break;
+    // default:
+    // Logger.recordMetadata("GitDirty", "Unknown");
+    // break;
+    // }
+
+    // map.setupLogging();
+    // if (!isReal()) {
+    // setUseTiming(false); // Run as fast as possible
+    // }
+    // // Start logging! No more data receivers, replay sources, or metadata values
+    // may
+    // // be added.
+    // Logger.start();
+
+    // led.colorAlliance().schedule();
+    // DriverStation.silenceJoystickConnectionWarning(true);
+
+    // }
```

Comment:
Seems to build with this enabled for me.  Likely need to add this function to the Alpha map.
```
    @Override
    public void setupLogging() {
        Logger.addDataReceiver(new WPILOGWriter("/media/sda1/")); // Log to a USB stick
        Logger.addDataReceiver(new NT4Publisher()); // Publish data to NetworkTables
        Logger.recordMetadata("RobotMap", this.getClass().getSimpleName());
        new PowerDistribution(1, ModuleType.kRev); // Enables power distribution logging
    }
```